cc/mcts_node.cc

```diff
@@ -65,6 +65,38 @@ MctsNode::MctsNode(MctsNode* parent, Coord move)
       stats(&parent->edges[move]),
       move(move),
       position(parent->position) {
+  // TODO(tommadams): move this code into the MctsPlayer and only perform it
+  // only if we are using an inference cache.
+  if (parent->HasFlag(Flag::kHasCanonicalSymmetry)) {
+    SetFlag(Flag::kHasCanonicalSymmetry);
+    canonical_symmetry = parent->canonical_symmetry;
+  } else {
+    // TODO(tommadams): skip this check if `move` is kPass or on the diagonal.
+    static_assert(symmetry::kIdentity == 0, "kIdentity must be 0");
+    auto best_symmetry = symmetry::kIdentity;
```

Comment:
a comment for what this is doing is nice.  "best" & "worst" should have their value system elucidated.

Reply:
done.